packages/react-core/package.json

```diff
@@ -48,7 +48,7 @@
     "@types/react-dom": "^17.0.13",
     "@types/react-test-renderer": "^17.0.1",
     "eslint": "^8.4.1",
-    "eslint-config-amplify-ui": "0.0.0",
+    "@aws-amplify/eslint-config-amplify-ui": "0.0.0",
```

Comment:
Can we alphabetize the impacted package `@aws-amplify/eslint-config-amplify-ui` deps in a follow up please?